plottr/node/data_selector.py

```diff
@@ -95,14 +96,14 @@ def selectedData(self) -> List[str]:
 
     @selectedData.setter  # type: ignore[misc]
     @updateOption('selectedData')
-    def selectedData(self, val: List[str]):
+    def selectedData(self, val: List[str]) -> None:
         if isinstance(val, str):
             val = [val]
         self._selectedData = val
 
     # Data processing
 
-    def validateOptions(self, data):
+    def validateOptions(self, data: Optional[Any]) -> bool:
```

Comment:
Note that the types of data should be improved. That will happen in a subsequent pr